document-portal/document-portal-fuse.c

```diff
@@ -3059,14 +3117,22 @@ static struct fuse_lowlevel_ops xdp_fuse_oper = {
  .fallocate    = xdp_fuse_fallocate,
 };
 
-static gpointer
-xdp_fuse_mainloop (gpointer data)
+typedef struct {
+  GMutex lock;
+  GCond cond;
+  struct fuse_session *se;
```

Comment:
It doesn't look like this field is being used. Can it be removed?

Reply:
Yeah, I noticed the leftover when I was about to push it, removed.